packages/x-adapter-platform/src/__tests__/__fixtures__/recommendations.response.ts

```diff
@@ -2,26 +2,31 @@ export const platformRecommendationsResponse = {
   topclicked: {
     content: [
       {
-        image: 'https://assets.empathy.co/images-demo/31335.jpg',
+        __images: ['https://assets.empathy.co/images-demo/31335.jpg'],
```

Comment:
Be careful with this, it seems that the top-clicked response is not standardized. We are going to wait until the response will be standarized.